core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/helpers/CsvHelper.java

```diff
@@ -0,0 +1,36 @@
+package com.exadel.etoolbox.linkinspector.core.services.helpers;
+
+import com.exadel.etoolbox.linkinspector.core.models.ui.GridViewItem;
+import com.exadel.etoolbox.linkinspector.core.services.data.models.GridResource;
+import org.apache.sling.api.resource.ResourceResolver;
+
+import java.util.List;
+
+public interface CsvHelper {
+
+    /**
+     * Reading items from csv and adapt them to the {@link GridResource}
+     *
+     * @param resourceResolver - {@link ResourceResolver}
+     * @param page             - page number of csv report
+     * @return the {@link List<GridResource>} representing the List of extracted grid resources form csv
+     */
+    List<GridResource> readCsvReport(ResourceResolver resourceResolver, int page);
+
+    /**
+     * Creating csv reports by list of {@link GridViewItem}. Reports are splitting on parts and used as pages.
```

Comment:
Same as above + "Reports are splittED IN parts". Please consider using a spellchecker for the whole text. "Grammarly" maybe... or some ChatGPT or something of that kind

Reply:
Fine. A good tip is to use ChatGPT.